### DISCUSSION TITLE
NRF24L01 non blocking recieve via interrupt

### DISCUSSION THREAD

**Discussion by derungebundener** [jgromes/RadioLib#365] [Q&A]
Need help with this module.
In exampe we can see a cooment - "See example ReceiveInterrupt for details" but there is no other example.

How can i read data without blocking?

Thanks.

**Comment by jgromes** [comment]
Hmm, looks like the interrupt examples for nRF24 are indeed missing. I added them in the latest commit.

**Reply by derungebundener** [reply]
Thanks so much, i will wait :)

**Reply by jgromes** [reply]
It was added yesterday, see https://github.com/jgromes/RadioLib/blob/master/examples/nRF24/nRF24_Receive_Interrupt/nRF24_Receive_Interrupt.ino
